CITATION.cff

```diff
@@ -0,0 +1,11 @@
+cff-version: 1.2.0
+message: "If Iris played an important part in your research then please add us to your reference list by using the references below."
+title: "Iris"
+authors:
+    - name: "Iris contributors"
```

Comment:
I don't want to block this PR with a load of work, so I'm happy to merge as-is.

However I do find the [ORCID](https://orcid.org/) approach quite appealing, especially since we get contributions from scientists who will already have a 'cited presence' in the world. So I have raised #5640.